hnn_core/gui/_viz_manager.py

```diff
@@ -431,8 +449,12 @@ def _get_ax_control(widgets, data, fig_idx, fig, ax):
         style=analysis_style,
     )
 
+    tagert_names = simulation_names[0:-1]
+    if len(simulation_names) > 1:
+        tagert_names = simulation_names[1:]
```

Comment:
Not sure I'm following here. It gets rid of the first simulation there is more than one set of data? Why?

Reply:
There is a bug at creating a new plot using the "Make figure" button. The "data to compare" dropdown doesnt show all the available datasets to compare. They only show up when the value of the "Simulation Data" dropdown changes (a callback function is invoked and updates both lists). These lines initialize  the list of available options without having to change the dropdowns values.